docs/home/api-info/webhooks.mdx

```diff
@@ -38,14 +39,18 @@ different messages when they become available - specifically:
 
 ## Authentication
 
-Validate requests to your public endpoint by comparing the webhook key with each request's HTTP header.
+When Metriport sends a webhook event, it includes an `x-metriport-signature` header. 
+
+- `x-metriport-signature`: This is a [HMAC](https://en.wikipedia.org/wiki/HMAC) SHA256 hash computed using your webhook key and the body of the webhook event.
+
+You can use this header to verify that the webhook events sent to your endpoint are from Metriport. Here's an example of how you can do this in Node.js:
```

Comment:
I know I sent this in a DM, but my comment about this was not addressed:

"it would be good to give explicit instructions on how to verify the signature in general. I'd be left thinking, okay, how do I do this in python?

ie generalize the instructions to compute the signature and verify outside of the Metriport TS SDK"

it's not obvious what `verifyWebhookSignature()` does without a dev looking at the implementation

Reply:
I thought about this as well, but then I looked at the HMAC link on Wikipedia, this is a standard algorithm, and Wikipedia also has a pseudocode to implement it. In practice, its likely that most programming languages have it built-in or through libraries.

Its like saying something its a MD5, there's a standard algorithm for it.